docs/static/config-management.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[config-management]]
-== Configuration Management
+== Managing Logstash
```

Comment:
@dedemorton should the URL resource (double brackets above) be changed to "management" instead?

Reply:
@acchen97 It's better not to change the ID because it's used for linking and HTML file names. If we change the ID, we have to ask the web team to set up redirects and we have to update all the references in the docs. And possibly backport the change so switching between releases in the docs is seamless (that means even more redirects). :-P Definitely should avoid changing IDs unless we have a problem to fix (like exposing old product names...that sort of thing).

Reply:
understood :)